models/Stable-diffusion/readme.md

```diff
@@ -0,0 +1,7 @@
+ * You have to install `checkpoints` models in this folder for `Stable Diffusion`.
```

Comment:
```suggestion
 * Place `*.ckpt` model files in this folder.
```

Reply:
Fixed in https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/1375/commits/e35521f36684498cccd6aec4bcee50b8156fc989